app/helpers/passengers_helper.rb

```diff
@@ -2,8 +2,10 @@ module PassengersHelper
   def expiration_check(passenger)
     if passenger.will_expire_within_a_week?
       'will_expire_soon'
-    elsif passenger.expired_within_3_days?
-      'expired_3_days'
+    elsif passenger.expired_within_grace_period?
+      'expired_within_grace_period'
+    elsif passenger.expired_for_3_days?
+      'expired_for_3_days'
```

Comment:
While you're doing that, the 7 days for the warning would also probably be a good candidate for semi-configurability.